jcl/src/java.base/share/classes/java/lang/invoke/MethodHandleResolver.java

```diff
@@ -307,7 +373,14 @@ private static final MethodHandle sendResolveMethodHandle(
 			String typeDescriptor,
 			ClassLoader loader) throws Throwable {
 		try {
+/*[IF OPENJDK_METHODHANDLES]*/
+			/* OpenJDK Lookup does not have a constructor Lookup(Class<?>, boolean). So, it has been replaced
+			 * with Lookup(Class<?>). */
+			MethodHandles.Lookup lookup = new MethodHandles.Lookup(currentClass);
+/*[ELSE] OPENJDK_METHODHANDLES*/
 			MethodHandles.Lookup lookup = new MethodHandles.Lookup(currentClass, false);
+/*[ENDIF] OPENJDK_METHODHANDLES*/
```

Comment:
The `false` indicates the Lookup object should skip the `package check`.  How does the equivalent behaviour get exposed in the OJDK Lookups?

Reply:
OpenJ9 does the package check for `java.lang.invoke.` (IAE) in the constructor, and it depends on the user input.

In OpenJDK, the package check is only done for the below methods before creating a new Lookup instance, and no user input is required:
```
- public Lookup in(Class<?>)
- public static Lookup privateLookupIn(Class<?>, Lookup)
- public Lookup dropLookupMode(int)
```
OpenJDK does handle the package check.

Reply:
Does this pass all the required tests?

Reply:
The builds with full testing haven't been enabled yet. I have added this as a TODO in https://github.com/eclipse/openj9/issues/10913. Once the builds are enabled, I will confirm this. Based upon the available OpenJDK Lookup constructors and micro-test results, `new MethodHandles.Lookup(Class<?>)` (in OpenJDK) should be identical to `new MethodHandles.Lookup(Class<?>, boolean)` (in OpenJ9).